src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -47,5 +47,8 @@ public static class EnvironmentSettingNames
         public const string AzureWebsiteAltZipDeployment = "WEBSITE_RUN_FROM_ZIP";
         public const string AzureWebsiteRunFromPackage = "WEBSITE_RUN_FROM_PACKAGE";
         public const string RegionName = "REGION_NAME";
+
+        public const string AzureAppServiceStorage = "WEBSITES_ENABLE_APP_SERVICE_STORAGE";
+        public const string CoreToolsEnvironment = "FUNCTIONS_CORETOOLS_ENVIRONMENT";
```

Comment:
Is CoreToolsEnvironment needed? Is it not enough to check if running in Azure or not?

Reply:
There is also other places like kubernetes, on devices and other possible future platforms. This setting is specific to core tools, so doing this way keeps the logic valid even when new platforms are introduced.